[Fix-11217] [Doc] add postgresql config in doc: datasource-setting (#11326)
apache/dolphinscheduler: docs/docs/en/guide/howto/datasource-setting.md

## Pseudo-Cluster/Cluster Initialize the Database

DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler

> which is `api-server/libs/` and `alert-server/libs` and `master-server/libs` and `worker-server/libs`, just ignore this steps if you use PostgreSQL. Let's walk through how to initialize the database in MySQL

Reviewer: How about adding note syntax for mysql part?

```md
NOTE: If you use MySQL, you need to manually download [mysql-connector-java driver][mysql] (8.0.16) and move it to the libs directory of DolphinScheduler ....
```

Author: oh, I know, let me change it.

Author: changed, please recheck, thanks.

## Pseudo-Cluster/Cluster Initialize the Database

> DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. Let's walk through how to initialize the database in MySQL and PostgreSQL :

Reviewer 2: ```suggestion
DolphinScheduler stores metadata in `relational database`. Currently, we support `PostgreSQL` and `MySQL`. Let's walk through how to initialize the database in `MySQL` and `PostgreSQL` :
```

> DolphinScheduler metadata is stored in the relational database. Currently, supports PostgreSQL and MySQL. Let's walk through how to initialize the database in MySQL and PostgreSQL :

Reviewer 2: In tech writing, we usually avoid `passive voice` unless necessary.

Author: > In tech writing, we usually avoid `passive voice` unless necessary.

OK, thanks for your advice, keep this rule in the future.

mysql> FLUSH PRIVILEGES;
``` 

For PostgreSQL: 
```shell
# Use psql-tools to login PostgreSQL
psql
# Create a database
postgres=# CREATE DATABASE dolphinscheduler;

> # Replace {user} and {password} with your username and password

Reviewer: I think we should use `<>` for the required parameter, could you please also change the other place?
```suggestion
# Replace <user> and <password> with your username and password
```

Author: OK, got it.

Author: Hi there, is it a bit strange for changing postresql part only? I think '{}' is clear for others, let's keep it or talk?
